manifest.yaml

```diff
@@ -24,6 +24,6 @@ configs:
 apimlServices:
   static:
   - file: apiml-static-registration.yaml.template
-desktopIframePlugins:
-- icon: app/img/explorer-JES.png
+desktopPlugins:
+- icon: web/img/explorer-JES.png
```

Comment:
If we convert this from `desktopIframePlugins` to `desktopPlugins`, the only parameter it takes is the file path to where `pluginDefinition.json` located. The manifest ppt is just converted to a PR for v1.19 release, which you can find out here https://github.com/zowe/docs-site/pull/1526/files#diff-388b86598a43590a411488b297bc9583b1fcf194eb552bca1934351c23852dd7R79.

Reply:
modified to 
```
appfwPlugins:
- path: "."
```